src/docker.test.ts

```diff
@@ -90,17 +131,17 @@ describe("Docker images", (): void => {
     expect(docker.CACHE_HIT).toBe("cache-hit");
   });
 
+  test("exports DOCKER_IMAGES_LIST", (): void => {
+    expect(docker.DOCKER_IMAGES_LIST).toBe("docker-images-list");
+  });
+
   test("exports DOCKER_IMAGES_PATH", (): void => {
     expect(docker.DOCKER_IMAGES_PATH).toBe("~/.docker-images.tar");
   });
 
   test("are loaded on cache hit", async (): Promise<void> => {
     await mockedLoadDockerImages(true);
 
-    expect(util.execBashCommand).lastCalledWith(
```

Comment:
Analogous to `"aren't saved on cache miss when new Docker image list is empty"`, it's probably worth asserting that we don't call `core.saveState` and that we only call `util.execBashCommand` once since we don't want to be needlessly listing the Docker images on cache hit.

Reply:
Your feedback re: calling `util.execBashCommand` once makes sense to me.  However, I think that we do call `core.saveState` even if there is a cache hit in the production function.  Check this out (from `docker.ts`:
```const loadDockerImages = async (): Promise<void> => {
  const requestedKey = getInput("key", { required: true });
  const restoredKey = await restoreCache([DOCKER_IMAGES_PATH], requestedKey);
  const cacheHit = requestedKey === restoredKey;
  saveState(CACHE_HIT, cacheHit);```
Given that, do you still want me to make an assert re: `core.saveState`?  Perhaps asserting that is only called once? Let me know what you think makes sense or if I'm missing something.

Reply:
Good call; yeah, we should simply assert that it's called once.

Reply:
I think there were some function calls missing from the `assertCalledInOrder` call and that's why it was not clear that `core.saveState` is called within `loadDockerImages` even when there is a cache hit.  I fixed that and am tentatively pushing the result.

Reply:
I can't think of any plausible counterexamples; I'll delete those function calls.